clang/lib/Driver/ToolChains/Arch/LoongArch.cpp

```diff
@@ -127,6 +127,11 @@ void loongarch::getLoongArchTargetFeatures(const Driver &D,
                                            const llvm::Triple &Triple,
                                            const ArgList &Args,
                                            std::vector<StringRef> &Features) {
+  // Enable the `lsx` feature on 64-bit LoongArch by default.
+  if (Triple.isLoongArch64() &&
+      (!Args.hasArg(clang::driver::options::OPT_march_EQ)))
```

Comment:
Prefer `hasArgNoClaim` to have a single place to claim the option.
Unclaimed options will lead to an unused argument warning.

Reply:
Updated to use `hasArgNoClaim` here.